<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalStripEditor</class>
 <widget class="QWidget" name="TerminalStripEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="m_tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_layout_tab">
      <attribute name="title">
       <string>Disposition</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="m_table_widget"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="8" column="1" colspan="2">
           <widget class="QComboBox" name="m_led_cb">
            <item>
             <property name="text">
              <string>Sans</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Avec</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_move_to_pb">
            <property name="toolTip">
             <string>Effectuer le déplacement</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../qelectrotech.qrc">
              <normaloff>:/ico/16x16/dialog-ok.png</normaloff>:/ico/16x16/dialog-ok.png</iconset>
            </property>
           </widget>
          </item>
          <item row="12" column="1" colspan="2">
           <widget class="ColorComboBox" name="m_bridge_color_cb"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Étage :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Type :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Couleur pont :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QComboBox" name="m_type_cb">
            <item>
             <property name="text">
              <string>Générique</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fusible</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sectionnable</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Diode</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Terre</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>LED :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Fonction :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="m_move_to_label">
            <property name="text">
             <string>Déplacer dans :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QComboBox" name="m_function_cb">
            <item>
             <property name="text">
              <string>Générique</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phase</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Neutre</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="m_move_to_cb">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QSpinBox" name="m_level_sb">
            <property name="maximum">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="m_auto_ordering_pb">
            <property name="text">
             <string>Position automatique</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QPushButton" name="m_group_terminals_pb">
            <property name="text">
             <string>Grouper les bornes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QPushButton" name="m_ungroup_pb">
            <property name="text">
             <string>Degrouper les bornes</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1" colspan="2">
           <widget class="QPushButton" name="m_bridge_terminals_pb">
            <property name="text">
             <string>Ponter les bornes</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1" colspan="2">
           <widget class="QPushButton" name="m_unbridge_terminals_pb">
            <property name="text">
             <string>Déponter les bornes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_data_tab">
      <attribute name="title">
       <string>Propriétés</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Commentaire :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Installation :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_location_le"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_installation_le"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_comment_le"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Localisation :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_name_le"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="m_description_te"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>../../colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../qelectrotech.qrc"/>
 </resources>
 <connections/>
</ui>
